### PR TITLE
Pranita_Asignment_19_Done_30min+10min+10min+10min+10min

### PR DESCRIPTION
**Pull request by PranitaPJun20**
allprime
firstdiffindex
alldiffindex
max
string

### CODE REVIEW

File: src/pranitaPJun20/AllPrimeNumber.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+			}
+		}
+		if(flag==true) {
+			System.out.println("Given number "+num+" is prime");
+	
+		}	
+	}
+	
+//Method to check Count,Sum and average of all prime numbers in range
+	boolean checkPrimeNumbRange(int num) {
```
Comment (KrishnaKTechnocredits) [inline src/pranitaPJun20/AllPrimeNumber.java]: Where is the range Pranita ??

File: src/pranitaPJun20/AllPrimeNumber.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+		return flag;
+	}
+	
+	public static void main(String[] args) {
+		AllPrimeNumber allPrimeNumber = new AllPrimeNumber();
+		int num =15;
+		allPrimeNumber.checkPrime(num);
+		int sum=0;
+		int count=0;
+		for(int num1=2;num1<=100;num1++) {
```
Comment (KrishnaKTechnocredits) [inline src/pranitaPJun20/AllPrimeNumber.java]: Clearly, said that you are not going to write any logic in the main method then ???

File: src/pranitaPJun20/CompareArrayMaxValueAtDiffIndex.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+		int[] arr2 = {10,2,18,14,3};
+		
+		CompareArrayMaxValueAtDiffIndex CompareArrayMaxValueAtDiffIndex = new CompareArrayMaxValueAtDiffIndex();
+		CompareArrayMaxValueAtDiffIndex.findMaxValueDifferIndex(arr1,arr2);
+	}
+	
+	void findMaxValueDifferIndex(int[] input1,int[] input2) {
+		
+		if(input1.length==input2.length) {
+			int max=0;
```
Comment (KrishnaKTechnocredits) [inline src/pranitaPJun20/CompareArrayMaxValueAtDiffIndex.java]: No Need to initialize here, you can directly initialize at line number 26.

**Review by KrishnaKTechnocredits — CHANGES_REQUESTED** [review]
.

File: src/pranitaPJun20/AllPrimeNumber.java (hunk 1 above)
Comment (PranitaPJun20) [reply src/pranitaPJun20/AllPrimeNumber.java]: Done .Changed the name of method.

File: src/pranitaPJun20/AllPrimeNumber.java (hunk 2 above)
Comment (PranitaPJun20) [reply src/pranitaPJun20/AllPrimeNumber.java]: Written logic in method now.

File: src/pranitaPJun20/CompareArrayMaxValueAtDiffIndex.java (hunk 3 above)
Comment (PranitaPJun20) [reply src/pranitaPJun20/CompareArrayMaxValueAtDiffIndex.java]: Done.

File: src/pranitaPJun20/AllPrimeNumber.java [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+			System.out.println("Given number " + num + " is prime");
+		}
+	}
+	// Method to check Count,Sum and average of all prime numbers in range
+	void checkPrimeNumb() {
+		int sumOfPrimeNum = 0;
+		int countOfPrimeNumb = 0;
+		for (int num = 2; num <= 100; num++) {
+			boolean flag = true;
+			for (int index = 2; index <= num / 2; index++) {
```
Comment (KrishnaKTechnocredits) [inline src/pranitaPJun20/AllPrimeNumber.java:33]: You should use isPrime() method here, np Take care next time. Merging code.

**Review by KrishnaKTechnocredits — APPROVED** [review]

**Pull request merged**